stargate-starter/src/test/java/io/stargate/starter/StarterTest.java

```diff
@@ -171,4 +171,69 @@ void testDeveloperModeSetsDefaultSeedsAndSnitch() {
     assertThat(System.getProperty("stargate.developer_mode")).isEqualTo("true");
     assertThat(System.getProperty("stargate.snitch_classname")).isEqualTo("SimpleSnitch");
   }
+
+  @Test
+  void testSetStargatePropertiesWithIPv4ListenHost() {
+    Starter starter = new Starter();
+    starter.simpleSnitch = true;
+    starter.seedList = Arrays.asList("127.0.0.1", "127.0.0.2");
+    starter.listenHostStr = "127.0.0.1";
+    starter.clusterName = "foo";
+    starter.version = "3.11";
```

Comment:
Readibility nit: collapse into a private method to remove copypasta and make tests read a bit quicker?